packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -157,10 +135,7 @@ export class LitAuthClient {
     // Set RPC URL
     this.rpcUrl =
       options?.rpcUrl ||
-      this.litNodeClient.config.litNetwork === 'datil-dev' ||
-      this.litNodeClient.config.litNetwork === 'datil-test'
-        ? LIT_RPC.VESUVIUS
-        : LIT_RPC.CHRONICLE;
+      RPC_URL_BY_NETWORK[this.litNodeClient.config.litNetwork];
```

Comment:
nit: though we have type safety there is a chance a user passes an invalid `network` param and this look up will fail. We might want to add a check after lookup to assure `this.rpcUrl` is defined.